### PR TITLE
Add MARO Blockchain

### PR DESCRIPTION
**Pull request by TTCOffical**

### CODE REVIEW

File: _data/chains/eip155-8848.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  "infoURL": "https://ma.ro/",
+  "shortName": "maro",
+  "chainId": 8848,
+  "networkId": 8848,
+  "explorers": [
+    {
+      "name": "MARO Scan",
+      "url": "https://scan.ma.ro/#/",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-8848.json:19]:
```suggestion
      "standard": "none"
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by Marvin8122 — APPROVED** [review]

**Review by 13364174888 — APPROVED** [review]

**Comment by 13364174888** [comment]
> 

好

**Review by 13364174888 — APPROVED** [review]
好

File: _data/chains/eip155-8848.json (hunk 1 above)
Comment (13364174888) [reply _data/chains/eip155-8848.json]: 我

File: _data/chains/eip155-8848.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://ma.ro/",
+  "shortName": "maro",
+  "chainId": 8848,
+  "networkId": 8848,
+  "explorers": [
+    {
+      "name": "MARO Scan",
+      "url": "https://scan.ma.ro/#/",
```
Comment (ligi) [inline _data/chains/eip155-8848.json:18]:
```suggestion
      "url": "https://scan.ma.ro/#",
```

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Comment by TTCOffical** [comment]
Hi, thanks for reviewing for our chains, we just delete the icon firstly and process it later, thanks

**Review by Marvin8122 — APPROVED** [review]

**Review by ligi — APPROVED** [review]

**Pull request merged**